lucille-core/src/test/java/com/kmwllc/lucille/test/TestConfUtils.java

```diff
@@ -0,0 +1,51 @@
+package com.kmwllc.lucille.test;
+
+import com.kmwllc.lucille.core.Runner;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * This class is only meant for testing configs in the Lucille example module.
+ */
+
```

Comment:
extra space

Reply:
removed in [e5cdd5e](https://github.com/kmwtechnology/lucille/pull/196/commits/e5cdd5eb8515f9058aef636b86008f676f4b489b)